Common/Securities/Option/OptionPriceModels.cs

```diff
@@ -0,0 +1,213 @@
+        }
+
+        /// <summary>
+        /// Bjerksund and Stensland pricing engine for American options (1993) 
+        /// QuantLib reference: http://quantlib.org/reference/class_quant_lib_1_1_bjerksund_stensland_approximation_engine.html
+        /// </summary>
+        /// <returns>New option price model instance</returns>
+        public static IOptionPriceModel BjerksundStensland()
+        {
+            return new QLOptionPriceModel(process => new BjerksundStenslandApproximationEngine(process),
+                                           _underlyingVolEstimator,
+                                           _riskFreeRateEstimator,
+                                           _dividendYieldEstimator);
+        }
+
+        /// <summary>
+        /// Pricing engine for European vanilla options using integral approach. 
+        /// QuantLib reference: http://quantlib.org/reference/class_quant_lib_1_1_integral_engine.html
+        /// </summary>
+        /// <returns>New option price model instance</returns>
+        public static IOptionPriceModel Integral()
+        {
+            return new QLOptionPriceModel(process => new IntegralEngine(process),
+                                           _underlyingVolEstimator,
+                                           _riskFreeRateEstimator,
+                                           _dividendYieldEstimator);
+        }
+
+        /// <summary>
+        /// Pricing engine for European options using finite-differences. 
+        /// QuantLib reference: http://quantlib.org/reference/class_quant_lib_1_1_f_d_european_engine.html
+        /// </summary>
+        /// <param name="timeSteps">Number of steps in binomial tree</param>
+        /// <returns>New option price model instance</returns>
+        public static IOptionPriceModel CrankNicolsonFD(int timeSteps = 201)
+        {
+            PricingEngineFuncEx pricingEngineFunc = (symbol, process) =>
+                            symbol.ID.OptionStyle == OptionStyle.American ?
+                            new FDAmericanEngine(process, timeSteps, timeSteps - 1) as IPricingEngine:
+                            new FDEuropeanEngine(process, timeSteps, timeSteps - 1) as IPricingEngine;
+
+            return new QLOptionPriceModel(pricingEngineFunc,
+                                           _underlyingVolEstimator,
+                                           _riskFreeRateEstimator,
+                                           _dividendYieldEstimator);
+        }
+
+        /// <summary>
+        /// Pricing engine for vanilla options using binomial trees. Jarrow-Rudd model.
+        /// QuantLib reference: http://quantlib.org/reference/class_quant_lib_1_1_f_d_european_engine.html
+        /// </summary>
+        /// <param name="timeSteps">Number of steps in binomial tree</param>
+        /// <returns>New option price model instance</returns>
+        public static IOptionPriceModel BinomialJarrowRudd(int timeSteps = 201)
```

Comment:
Thanks for noticing. Forgot to hide this parameter. Will make it available together with other option pricing customizations (see above) later while making sure it is computationally cheap.